backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Inout_Details_Footer.sql

```diff
@@ -1,77 +1,79 @@
-		2			as pozition
-	FROM
-		m_inout io
-	WHERE io.isActive = 'Y'
-	)footer
-WHERE
-	footer.m_inout_id = $1
-	AND (footer.language = $2 OR footer.language IS NULL)
-	AND (textleft <> '' OR textcenter <> '')
-	AND (textleft IS NULL OR textcenter IS NULL)
-ORDER BY
-	pozition
+SELECT footer.*,
+       COALESCE(inc_trl.name, inc.name) AS Incoterms,
+       io.incotermlocation
+FROM (
+         --Docnote DE
+         SELECT NULL                                                                                AS textleft,
+                CASE
+                    WHEN io.descriptionbottom IS NOT NULL
+                        THEN E'\n\n\n'
+                        ELSE ''
+                END || dt.documentnote                                                              AS textcenter,
+                (SELECT AD_Language FROM AD_Language WHERE IsBaseLanguage = 'Y' AND isActive = 'Y') AS language,
+                io.m_inout_id                                                                       AS m_inout_id,
+                'docnote'                                                                           AS tag,
+                3                                                                                   AS pozition
+         FROM m_inout io
+                  LEFT JOIN c_doctype dt ON io.c_doctype_id = dt.c_doctype_id AND dt.isActive = 'Y'
+         WHERE io.isActive = 'Y'
+         UNION
+         ---------------------------------------------------------------------------------------------
+         --Docnote TRL
+         SELECT NULL                   AS textleft,
+                CASE
+                    WHEN io.descriptionbottom IS NOT NULL
+                        THEN E'\n\n\n'
+                        ELSE ''
+                END || dt.documentnote AS textcenter,
+                dt.ad_language         AS language,
+                io.m_inout_id          AS m_inout_id,
+                'docnote'              AS tag,
+                3                      AS pozition
+         FROM m_inout io
+                  LEFT JOIN c_doctype_trl dt ON io.c_doctype_id = dt.c_doctype_id AND dt.isActive = 'Y'
+         WHERE io.isActive = 'Y'
+         UNION
+         ---------------------------------------------------------------------------------------------
+         --Descriptionbottom
+         SELECT io.descriptionbottom AS textleft,
+                NULL                 AS textcenter,
+                NULL                 AS language,
+                io.m_inout_id        AS m_inout_id,
+                'descr'              AS tag,
+                2                    AS pozition
+         FROM m_inout io
+         WHERE io.isActive = 'Y') footer
+         INNER JOIN m_inout io ON io.m_inout_id = footer.m_inout_id
+         LEFT OUTER JOIN C_Incoterms inc ON io.c_incoterms_id = inc.c_incoterms_id
+         LEFT OUTER JOIN C_Incoterms_trl inc_trl ON inc.c_incoterms_id = inc_trl.c_incoterms_id AND inc_trl.ad_language = 'de_DE'
```

Comment:
Please, move this part in each union:
and use the parameter for language

Reply:
done